autoware_adapi_v1_msgs/vehicle/msg/VehicleDoor.msg

```diff
@@ -0,0 +1,9 @@
+# constants for door status
+uint8 UNKNOWN = 0
+uint8 DOOR_OPENED = 1
+uint8 DOOR_CLOSED = 2
+uint8 DOOR_OPENING = 3
+uint8 DOOR_CLOSING = 4
+uint8 NOT_APPLICABLE = 5
+
+uint8 status
```

Comment:
I heared that some vehicles have free/manual mode.

Reply:
corrected in https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/ffc28266e7b4b930dcaccfbc83044af191c0b085